src/Memento.jl

```diff
@@ -59,10 +60,12 @@ const _loggers = Dict{AbstractString, Logger}(
 )
 
 const LOGGER = getlogger(@__MODULE__)
+const _default_formatter = DefaultFormatter(DEFAULT_FMT_STRING)
 
 function __init__()
     Memento.config!(DEFAULT_LOG_LEVEL)
     Memento.register(LOGGER)
```

Comment:
this logger is never used, we can this delete this line
and line 62

Reply:
https://github.com/invenia/Memento.jl/blob/master/src/loggers.jl#L142

Reply:
ah i missed that one